### PR TITLE
Fix some of the build warnings

### PR DESCRIPTION
**Pull request by bgavrilMS**
This should fix all but 2 warnings: 

- Xamarin wanting to connect to a MAC 
- warning NU1603: runtime.native.System.IO.Compression 4.3.0

These are both external and I am tracking them

### CODE REVIEW

File: adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/netstandard1.1/PlatformParameters.cs [hunk 1]
```diff
@@ -30,9 +30,15 @@
 
 namespace Microsoft.IdentityModel.Clients.ActiveDirectory
 {
+    /// <summary>
+    ///  Additional parameters used in acquiring user's authorization, specific to netstadard 1.1
```
Comment (jennyf19) [inline adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/netstandard1.1/PlatformParameters.cs:34]: `netstandard 1.1` needs an "n"  here and in 1.3 file. Thanks for adding this (I should have done this when I did the webview changes)

**Review by jennyf19 — APPROVED** [review]
🚀

**Comment by jennyf19** [comment]
@bgavrilMS Where are you tracking the work items for these? VSTS?

**Comment by bgavrilMS** [comment]
Merged manually

**Pull request closed without merging**